implementations/swift/ockam/ockam_app/Ockam/IgnoreService.swift

```diff
@@ -0,0 +1,56 @@
+import SwiftUI
+
+struct IgnoreServiceView: View {
+    @Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
+    @State var service: Service
+
+    var body: some View {
+        VStack {
+            Text(
+                "Once you click Ignore, the service '\(service.sourceName)' will no longer show up here.\n" +
+                "Are you sure you want to do this?\n" +
+                "Once ignored, the only way to get this back is to have the person who sent you the invite, to send another one."
```

Comment:
Perhaps this could be reworded/reorganized a little, for example:

```
"Are you sure you want to do this?\n"}
"The service '\(service.sourceName)' you will no longer have access to it and you will have to request a new invite to the service's owner."
```